lisp/ess-custom.el

```diff
@@ -2321,41 +2325,88 @@ and `ess-tracebug'.")
 
 ;;*;; Inferior ESS commands
 
-(defvar ess-load-command "source(\"%s\")\n"
+(defvar ess-load-command "source('%s')\n"
   "Dialect specific format-string for building the ess command to load a file.
 
 This format string should use %s to substitute a file name and should
 result in an ESS expression that will command the inferior ESS to load
 that file.")
-(define-obsolete-variable-alias 'inferior-ess-load-command 'ess-load-command "ESS v13.09")
-
-(defvar ess-load-visibly-command nil
```

Comment:
>> On Mon, Apr 04 2016 14:51, Lionel Henry wrote:

> Yes it would work but it would be less flexible I think.

Probably so, but it's easier for the user to modify. There might be that the
current format commands are already modified by some users.

Quite frankly I mind the most the excessive layers of indirection. Function ->
format string -> low level function (.ess.eval, .ess.source etc).

> That could be a good idea, but would it make things less clear? How would we
> call this function?

We use terms source, load and eval somewhat interchangeably. How about
*-eval-formatter?  The long name ess-format-eval-command is not very appealing
to me because 'format' reads as verb in there.


  Vitalie